pkg/controller/node.go

```diff
@@ -225,7 +225,7 @@ func (c *Controller) handleAddNode(key string) error {
 	nodeAddr := util.GetNodeInternalIP(*node)
 	for _, ip := range strings.Split(ipStr, ",") {
 		if util.CheckProtocol(nodeAddr) == util.CheckProtocol(ip) {
-			err = c.ovnClient.AddStaticRoute("", nodeAddr, ip, c.config.ClusterRouter)
+			err = c.ovnClient.AddStaticRoute("", nodeAddr, ip, c.config.ClusterRouter, false)
```

Comment:
use const or type to define use ecmp or not for better understanding

Reply:
modified